include/bl_otp_counters.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BL_OTP_COUNTERS_H_
+#define BL_OTP_COUNTERS_H_
+
+#include <string.h>
+#include <zephyr/types.h>
+#include <drivers/nrfx_common.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
This commit is too big.

Moving bl_otp_counters out of bl_storage should be in it's own commit.

Reply:
(and this commit should not contain changes (e.g. COUNTER_DESC_NSIB_HW_COUNTER_VERSION)).